pbrd/pbr_zebra.c

```diff
@@ -516,7 +516,7 @@ pbr_encode_pbr_map_sequence_vrf(struct stream *s,
 		return;
 	}
 
-	stream_putl(s, pbr_vrf->vrf->data.l.table_id);
+	stream_putl(s, pbr_vrf->vrf->data.l.local_table_id);
```

Comment:
why not avoid changing the vrf_data struct, and just put the ifdef block right here, if this is the only place that needs to use the special netlink value?